### PR TITLE
Added a 'is_ups_worldship_shipping' a function field

### PR DESCRIPTION
**Pull request by tarunbhardwaj**

### CODE REVIEW

File: stock.py [hunk 1]
```diff
@@ -37,7 +37,11 @@ class ShipmentOut:
     __name__ = 'stock.shipment.out'
 
     is_ups_shipping = fields.Function(
-        fields.Boolean('Is Shipping', readonly=True),
+        fields.Boolean('Is UPS Shipping', readonly=True),
+        'get_is_ups_shipping'
+    )
+    is_ups_worldship_shipping = fields.Function(
+        fields.Boolean('Is UPS Worldship Shipping', readonly=True),
```
Comment (priyankarani) [inline stock.py]: must have question mark at end of string

File: stock.py [hunk 2]
```diff
@@ -80,7 +84,7 @@ def get_is_ups_shipping(self, name):
         """
         Check if shipping is from UPS
         """
-        return self.carrier and self.carrier.carrier_cost_method == 'ups'
+        return self.carrier and self.carrier.carrier_cost_method == name[3:-9]
```
Comment (priyankarani) [inline stock.py]: can you add tests for this ?

File: stock.py [hunk 3]
```diff
@@ -37,7 +37,11 @@ class ShipmentOut:
     __name__ = 'stock.shipment.out'
 
     is_ups_shipping = fields.Function(
-        fields.Boolean('Is Shipping', readonly=True),
+        fields.Boolean('Is UPS Shipping', readonly=True),
```
Comment (priyankarani) [inline stock.py]: function field is already readonly

File: stock.py (hunk 2 above)
Comment (tarunbhardwaj) [inline stock.py]: done

**Comment by priyankarani** [comment]
LGTM

**Pull request merged**